tests/data/format/no_whitespace_stripper/module_docstring.py.out

```diff
@@ -0,0 +1,18 @@
+"""  A multi-line  .
+
+docstring  
+"""
+
+"""  A multi-line  .
+
+docstring  
+   
+"""
+
+"""  My docstring.
+
+ 
+    
+   My indented section  
+  
```

Comment:
This seems to be a very weird edge case. Shouldn't L17 have as many whitespaces as L12 has in the original file?